src/test/java/org/llorllale/cactoos/matchers/ThrowsTest.java

```diff
@@ -126,8 +126,36 @@ public void describeExpectedValues() {
             "describes the expected exception",
             description.toString(),
             new IsEqual<>(
-                "Exception has type 'java.lang.NullPointerException'"
-                    + " and message 'NPE'"
+                "Exception has type <class java.lang.NullPointerException>"
+                    + " and message matches \"NPE\""
+            )
+        ).affirm();
+    }
+
+    @Test
+    public void noMessageMatchesException() {
+        new Assertion<>(
+            // @checkstyle LineLength (1 line)
+            "must match if message is no present",
```

Comment:
@victornoel This line doesn't seem to be long enough to require checkstyle comment.